tests/daemon/test-persistent-cache.c

```diff
@@ -884,13 +884,12 @@ test_persistent_cache_builds_boot_metadata_file (gboolean     *unused,
 
   EmerPersistentCache *cache = make_testing_cache ();
 
-  gint64 first_offset = read_offset ();
+  gint64 offset = read_offset ();
 
   g_assert_true (boot_timestamp_is_valid (relative_time, absolute_time));
-  gint64 second_offset = read_offset();
 
   // The offset should not have changed.
```

Comment:
This comment has now fallen out of sync with the implementation. We can probably just remove it.